swagger.yaml

```diff
@@ -622,6 +622,17 @@ definitions:
         description: The date that the employee accepted or denied to have their salary information shared.
       retirementCode:
         type: string
+      i9FormCode:
```

Comment:
this field will rarely be used and there isn't a validation table for the various codes :(. We can always add a `i9FormDescription` field in the future if we clients want a more human readable value, but for now it would be nice to not have the added maintenance.

Reply:
For `i9FromCode`, is it possible to get all possible values? Also, is there isn't a validation table, do you know which table we can use to get the descriptions?

Reply:
@tsoliangwu0130 There isn't a validation table, so there isn't a table we can get the descriptions from :(